### DISCUSSION TITLE
Make it easier to disable subsystems

### DISCUSSION THREAD

**Discussion by ahjohannessen** [nordic-institute/X-Road#2171] [Ideas]
In order to make a provider Security Server invisible to other Security Servers, one needs to disable the subsystems that are used to publish services.  This is a nice feature, however if one has a lot of subsystems it would be nice with the ability to disable/enable all subsystems on a Security Server.

**Comment by raits** [comment]
Hello @ahjohannessen!

Sorry for the late reply, I have included this proposal in the next Technical Committee meeting taking place on the 19th of June.

**Comment by raits** [comment]
Hello @ahjohannessen,

I am happy to say that this proposal was accepted. We will look further into how to implement it in practice. Further progress can be followed in the backlog item: https://nordic-institute.atlassian.net/browse/XRDDEV-2657.
